Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:16.13.0-slim
+FROM node:18.15.0-slim
```

Comment:
Do we even use this `Dockerfile` anywhere? Can we remove this?

Reply:
🤔  Let me check and i'll remove it if not.

Reply:
It's used by Uffizzi.